src/components/obsTab/ObsTab.js

```diff
@@ -26,51 +36,116 @@ class General extends React.Component {
             messageType: 'geometry_msgs/Point'
         });
 
-        this.anglesTopic.subscribe((msg) => {
-            console.log(msg);
-            this.setState({
-                angles: {
-                    x: msg.x,
-                    y: msg.y,
-                }
-            });
+        this.dataTopic = new ROSLIB.Topic({
+            ros: this.ros,
+            name: 'obs/data',
+            messageType: 'std_msgs/Float64MultiArray'
         });
 
-        // TODO Remove this, just a demo
-        this.spammer = setInterval(() => this.anglesTopic.publish(this.newPoint()), 100);
+        this.voltageTopic = new ROSLIB.Topic({
+            ros: this.ros,
+            name: 'obs/voltage',
+            messageType: 'std_msgs/Float64'
+        });
+
+        this.anglesTopic.subscribe(this.anglesHandler);
+        this.dataTopic.subscribe(this.dataHandler);
+        this.voltageTopic.subscribe(this.voltageHandler);
+
+        // TODO Remove this, just for demo
```

Comment:
Remove the demo stuff, when we need to test this more we can use a seperate node for publishing dummy-data, or even better; use the actual hardware.